test/Observable_mockito_test.dart

```diff
@@ -0,0 +1,46 @@
+library wCommon.test;
+
+// Testing imports
+import 'package:unittest/unittest.dart';
+import 'package:mockito/mockito.dart';
+import 'package:unittest/html_enhanced_config.dart';
+
+// Testing dependencies
+import 'dart:async';
+import 'dart:js';
+import 'package:wCommon/wCommon.dart';
+
+class MockJsObject extends Mock {}
+
+main() {
+
+  useHtmlEnhancedConfiguration();
+
+  var observable;
+  var jsObservable;
+
+  group('Observable', () {
+
+    setUp(() {
+      jsObservable = new MockJsObject();
+
+      observable = new Observable(jsObservable);
+    });
+
+    test('it should return a stream for the stream property', () {
+      var stream = observable.stream;
+
+      expect(stream, new isInstanceOf<Stream>());
+    });
+
+    test('it should proxy calls to the dispatch method down to the jsObservable', () {
+      var params = [];
+      var callback = () {};
+
+      observable.dispatch(params, callback);
+
+      // This should be matching the arguments, but it is not.  Bug in Mockito?
+      verify(jsObservable.callMethod('dispatch', [params, callback])).called(1);
```

Comment:
The Mockito mocks are much less code to accomplish the same thing as stock.  Plus, the mocks are referenced as properties/methods, not strings, so IDE goodness is available for refactors, code completion, go to definition.  I would vote for using Mockito, unless the new test package radically changes stock mocks.